standalone/ansible/group_vars/load_balancing.yml

```diff
@@ -15,6 +15,7 @@ haproxy_cert_names:
   - /etc/ssl/simple.org
   - /etc/ssl/in.simple.org
   - /etc/ssl/bd.simple.org
+  - /etc/ssl/api-demo.et.simple.org
```

Comment:
Weird, looks like my earlier comment was published. Do we have to use a separate cert file here? Can we re-use the wildcard cert?

Reply:
The wildcard cert works for one-level subdomains, two-level wildcards like `*.*.simple.org` need their own. That's why we have certs for `in.` and `bd.` as well.

Reply:
Oh ouch. Sounds good! And we're not getting a `et.simple.org` cert because this is just a temp one, I guess?

Reply:
Ideally this should be the wildcard for `et.simple.org`, yes. We could buy it/setup certbot and then change this when that's done.

Reply:
Totally fine for now.